.github/workflows/actions/install-node/action.yml

```diff
@@ -11,7 +11,11 @@ runs:
       with:
         # Restore `node_modules` cache (unless packages change)
         key: npm-install-cache-${{ runner.os }}-${{ hashFiles('package-lock.json', '**/package.json') }}
-        path: node_modules
+        path: |
+          node_modules
+          app/node_modules
```

Comment:
We should ensure any (potential) workspace `node_modules` are also cached

Due to symbolic links created by [`npm link`](https://docs.npmjs.com/cli/v9/commands/npm-link) or [npm workspaces](https://docs.npmjs.com/cli/v9/using-npm/workspaces?v=true) we can't use:

```yaml
path: |
  **/node_modules
```

Otherwise we could match and cache "packages of linked packages" one day:

```md
app/node_modules/govuk-frontend/node_modules/** #symlink-1
node_modules/govuk-frontend/node_modules/** #symlink-1
package/node_modules/** #source
```